src/ts/header_generated.ts

```diff
@@ -5,12 +5,21 @@ import { flatbuffers } from 'flatbuffers'
  * @enum {number}
  */
 export enum GeometryType{
-  Point= 0,
-  MultiPoint= 1,
+  Point= 1,
   LineString= 2,
-  MultiLineString= 3,
-  Polygon= 4,
-  MultiPolygon= 5
+  Polygon= 3,
+  MultiPoint= 4,
+  MultiLineString= 5,
+  MultiPolygon= 6,
+  GeometryCollection= 7,
+  CircularString= 8,
+  CompoundCurve= 9,
+  CurvePolygon= 10,
+  MultiCurve= 11,
+  MultiSurface= 12,
```

Comment:
as a comment, perhaps add that 13 and 14 are reserved values for the abstract Curve and Surface types ? (so that they don't get accidentally reused for other purposes)

Reply:
Agreed, fixed.